Cargo.toml

```
@@ -17,7 +17,7 @@ members = [
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2021"
 version = "0.4.2"
-rust-version = "1.70.0"
+rust-version = "1.87.0"
```

I think this isn't need anymore.

```suggestion
rust-version = "1.70.0"
```